spec/feature_flagger/configuration_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+module FeatureFlagger
+  RSpec.describe Configuration do
+    context 'accessors methods' do
+      subject(:configuration) { described_class.new }
+
+      it { expect(configuration).to respond_to(:storage) }
+      it { expect(configuration).to respond_to(:storage=) }
+      it { expect(configuration).to respond_to(:yaml_filepath) }
+      it { expect(configuration).to respond_to(:info) }
+    end
```

Comment:
The configuration will still handle the yaml_path and info, just its initialization will be abstracted. But the user still can change it using `config.yaml_file_path = '...'`